packages/theming/theme-tokens/src/pipeline-input/token-input-light.macos.json

```diff
@@ -0,0 +1,134 @@
+{
+  "$schema": "https://raw.githubusercontent.com/microsoft/fluentui-token-pipeline/main/docs/token.schema.json",
+  "Meta": {
+    "FluentUITokensVersion": 0
+  },
+  "Global": {
```

Comment:
Since this is a global I would expect to see it in a global input file. Maybe move to the macos brand json file and just rename it to token-input-macos?

Reply:
Agreed, I'll move it to the global tokens file. Though I think we want to keep the filename as token-input-light.macos so that light / dark tokens are differentiated?

Reply:
Ah I meant you can move the font stuff to the token-input-brand.macos.json file, and then rename token-input-brand.macos to token-input.macos, or something, so that the file doesn't imply that it only holds brand information.

Reply:
gotcha! fixed in 39abe89c1689464cfce3c38bca793e021a0477b4